test/file.ts

```diff
@@ -2024,6 +2024,21 @@ describe('File', () => {
     });
   });
 
+  describe('deleteResumableCache', () => {
+    it('should delete resumable file upload cache', done => {
+      resumableUploadOverride = {
+        upload() {
+          return {
```

Comment:
Could you add assertions here that the correct parameters were passed? The lines we'd be testing from the source code:

```js
bucket: this.bucket.name,
file: this.name,
generation: this.generation,
```

Reply:
@stephenplusplus  added assertions.